yt_dlp/extractor/boosty.py

```diff
@@ -162,9 +165,19 @@ def _extract_formats(self, player_urls, video_id):
 
     def _real_extract(self, url):
         user, post_id = self._match_valid_url(url).group('user', 'post_id')
+
+        auth_cookie = self._get_cookies('https://boosty.to/').get('auth')
+        auth_headers = {}
+        if auth_cookie is not None:
+            try:
+                auth_data = json.loads(compat_urllib_parse_unquote(auth_cookie.value))
```

Comment:
```suggestion
                auth_data = json.loads(urllib.parse.unquote(auth_cookie.value))
```

change import as well

Reply:
Sure, didn't notice that's deprecated.